packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.js

```diff
@@ -4,14 +4,18 @@ import { forwardRef, useEffect, useRef } from 'react';
 import styled, { css, useTheme } from 'styled-components'
 import draggable from '../draggable'
 
-const STROKE_WIDTH = 2
-const SELECTED_STROKE_WIDTH = 4
+const STROKE_WIDTH = 3
+const SELECTED_STROKE_WIDTH = 6
```

Comment:
I've based these numbers on a `scale` of 0.67 but feel free to adjust them. 

These set the styles for all drawing tools except the transcription line and freehand line, each of which has its own custom styles.

Reply:
Also noting that I used the live Wildwatch Burrowing Owl project (PFE) as my reference for the stroke widths. The production classifier in the monorepo uses stroke widths that are much larger (for this image scale.)

PFE (live project):
<img width="800" alt="Screenshot of drawn rectangles for Wildwatch Burrowing Owl in Panoptes Front End." src="https://github.com/user-attachments/assets/f1e6045f-8699-4efb-a359-14764a5c06ac">


FEM:
<img width="800" alt="Screenshot of drawn rectangles for Wildwatch Burrowing Owl in Front End Monorepo." src="https://github.com/user-attachments/assets/9dbe1883-b937-4d21-8e56-a0119ffd22fc">

Reply:
If you'd prefer thicker drawing strokes for migrated projects, just adjust the numbers in the `Mark` component.

Reply:
After looking at I Fancy Cats, where the subject images are 640px, I've switched these from absolute to relative sizes, for consistent mark widths across SVG subjects with different viewports.